.github/workflows/chart_release.yaml

```diff
@@ -48,3 +48,58 @@ jobs:
           config: .github/cr.yaml
         env:
           CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+
+  gitops_update:
+    runs-on: ubuntu-latest
+    name: Update GitOps Repo
+    needs:
+      - chart-release
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Git
+        run: |
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+      - name: Add SSH key
+        env:
+          DITP_GITOPS_REPO_SECRET: ${{ secrets.DITP_GITOPS_REPO_SECRET }}
+        run: |
+          mkdir -p ~/.ssh
+          echo "$DITP_GITOPS_REPO_SECRET" > ~/.ssh/id_rsa
+          chmod 600 ~/.ssh/id_rsa
+          ssh-keyscan github.com >> ~/.ssh/known_hosts
+      - name: Clone gitops-repo
+        run: |
+          git clone git@github.com:bcgov-c/ministry-gitops-ditp.git
+          cd ministry-gitops-ditp
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+      - name: Lookup latest chart
+        id: chart_version
+        run: |
+          helm repo add traction https://bcgov.github.io/traction
+          helm repo update
+          echo "::set output name=APP_VERSION::$(helm search repo vc-authn-oidc --version ^0.3 -ojson | jq '.[0].app_version')"
+          echo "::set output name=CHART_VERSION::$(helm search repo traction --version ^0.3 -ojson | jq '.[0].version')"
```

Comment:
Is there a way to specify the version so that we do not need to keep this up-to-date manually?

Reply:
I'll remove the --version flag as discussed.